# HG changeset patch
# User Rik <rik@octave.org>
# Date 1382209550 25200
#      Sat Oct 19 12:05:50 2013 -0700
# Node ID dba2e06dcdb5b92621b35c88c488eed757a8c32c
# Parent  1be2993d3656e1cd30799765f2ad06cff72acc9c
maint: Move scripts/ui directory into scripts/gui directory.

* scripts/gui/errordlg.m, scripts/gui/helpdlg.m, scripts/gui/inputdlg.m,
scripts/gui/listdlg.m, scripts/gui/msgbox.m,
scripts/gui/private/message_dialog.m, scripts/gui/questdlg.m,
scripts/gui/warndlg.m: Moved from scripts/ui directory.

* scripts/gui/module.mk: Add scripts to build system.

* scripts/Makefile.am: Remove include for ui/module.mk

* scripts/ui/module.mk: Remove unused file.

diff --git a/scripts/Makefile.am b/scripts/Makefile.am
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -70,17 +70,16 @@ include special-matrix/module.mk
 include startup/module.mk
 include statistics/base/module.mk
 include statistics/distributions/module.mk
 include statistics/models/module.mk
 include statistics/tests/module.mk
 include strings/module.mk
 include testfun/module.mk
 include time/module.mk
-include ui/module.mk
 
 nobase_fcnfile_DATA = $(FCN_FILES) $(GEN_FCN_FILES) $(JAR_FILES)
 
 image_DATA = $(IMAGES)
 
 FCN_FILES_IN = $(GEN_FCN_FILES:.m=.in)
 
 if AMCOND_BUILD_DOCS
diff --git a/scripts/ui/errordlg.m b/scripts/gui/errordlg.m
rename from scripts/ui/errordlg.m
rename to scripts/gui/errordlg.m
diff --git a/scripts/ui/helpdlg.m b/scripts/gui/helpdlg.m
rename from scripts/ui/helpdlg.m
rename to scripts/gui/helpdlg.m
diff --git a/scripts/ui/inputdlg.m b/scripts/gui/inputdlg.m
rename from scripts/ui/inputdlg.m
rename to scripts/gui/inputdlg.m
diff --git a/scripts/ui/listdlg.m b/scripts/gui/listdlg.m
rename from scripts/ui/listdlg.m
rename to scripts/gui/listdlg.m
diff --git a/scripts/gui/module.mk b/scripts/gui/module.mk
--- a/scripts/gui/module.mk
+++ b/scripts/gui/module.mk
@@ -2,34 +2,42 @@ FCN_FILE_DIRS += gui
 
 gui_PRIVATE_FCN_FILES = \
   gui/private/__file_filter__.m \
   gui/private/__fltk_file_filter__.m \
   gui/private/__is_function__.m \
   gui/private/__uigetdir_fltk__.m \
   gui/private/__uigetfile_fltk__.m \
   gui/private/__uiobject_split_args__.m \
-  gui/private/__uiputfile_fltk__.m
+  gui/private/__uiputfile_fltk__.m \
+  gui/private/message_dialog.m
 
 gui_FCN_FILES = \
+  gui/errordlg.m \
   gui/guidata.m \
   gui/guihandles.m \
+  gui/helpdlg.m \
+  gui/inputdlg.m \
+  gui/listdlg.m \
+  gui/msgbox.m \
+  gui/questdlg.m \
   gui/uicontextmenu.m \
   gui/uicontrol.m \
   gui/uigetdir.m \
   gui/uigetfile.m \
   gui/uimenu.m \
   gui/uipanel.m \
   gui/uipushtool.m \
   gui/uiputfile.m \
   gui/uiresume.m \
   gui/uitoggletool.m \
   gui/uitoolbar.m \
   gui/uiwait.m \
   gui/waitbar.m \
   gui/waitforbuttonpress.m \
+  gui/warndlg.m \
   $(gui_PRIVATE_FCN_FILES)
 
 FCN_FILES += $(gui_FCN_FILES)
 
 PKG_ADD_FILES += gui/PKG_ADD
 
 DIRSTAMP_FILES += gui/$(octave_dirstamp)
diff --git a/scripts/ui/msgbox.m b/scripts/gui/msgbox.m
rename from scripts/ui/msgbox.m
rename to scripts/gui/msgbox.m
diff --git a/scripts/ui/private/message_dialog.m b/scripts/gui/private/message_dialog.m
rename from scripts/ui/private/message_dialog.m
rename to scripts/gui/private/message_dialog.m
diff --git a/scripts/ui/questdlg.m b/scripts/gui/questdlg.m
rename from scripts/ui/questdlg.m
rename to scripts/gui/questdlg.m
diff --git a/scripts/ui/warndlg.m b/scripts/gui/warndlg.m
rename from scripts/ui/warndlg.m
rename to scripts/gui/warndlg.m
diff --git a/scripts/ui/module.mk b/scripts/ui/module.mk
deleted file mode 100644
--- a/scripts/ui/module.mk
+++ /dev/null
@@ -1,20 +0,0 @@
-FCN_FILE_DIRS += ui
-
-ui_PRIVATE_FCN_FILES = \
-  ui/private/message_dialog.m
-
-ui_FCN_FILES = \
-  ui/errordlg.m \
-  ui/helpdlg.m \
-  ui/inputdlg.m \
-  ui/listdlg.m \
-  ui/msgbox.m \
-  ui/questdlg.m \
-  ui/warndlg.m \
-  $(ui_PRIVATE_FCN_FILES)
-
-FCN_FILES += $(ui_FCN_FILES)
-
-PKG_ADD_FILES += ui/PKG_ADD
-
-DIRSTAMP_FILES += ui/$(octave_dirstamp)
